lua/org-bullets.lua

```diff
@@ -88,7 +100,7 @@ end
 ---@param lastline number 'the previous last line'
 ---@param new_lastline number 'the updated last line'
 ---@param byte_count integer
-local function update_changed_lines(_, buf, __, firstline, lastline, new_lastline, byte_count)
+local function update_changed_lines(_, buf, _, firstline, lastline, new_lastline, byte_count)
```

Comment:
I don't think this change is necessary and actually leads to issues with `sumneko_lsp` basically the emmy lua annotation requires that parameters are named differently. Changing this now means that 2 parameters are now named `_` which causes a warning message. I think this should go back to what it was.

Reply:
Ah makes since, the change was based on the result of running `luacheck` on the file. luacheck sees `__` as an unused variable.